notebooks/Introduction.ipynb

```diff
@@ -216,7 +302,21 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "ename": "",
+     "evalue": "",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[1;31mJupyter cannot be started. Error attempting to locate jupyter: Running cells with 'Python 3.8.9 64-bit' requires notebook and jupyter package.\n",
+      "Run the following command to install 'jupyter and notebook' into the Python environment. \n",
+      "Command: 'python -m pip install jupyter notebook -U\n",
+      "or\n",
+      "conda install jupyter notebook -U'\n",
+      "Click <a href='https://aka.ms/installJupyterForVSCode'>here</a> for more info."
+     ]
+    }
+   ],
```

Comment:
These traceback additions seem out of place, no?

Reply:
Yup, shouldn't have been there. Fixed now